book/01-introduction/sections/02-installing.asc

```diff
@@ -66,15 +66,35 @@ Note that there are some limitations to portable mode:
 
 ==== Atom on Linux
 
-To install Atom on Linux, you can download a https://atom.io/download/deb[Debian package] or https://atom.io/download/rpm[RPM package] either from the https://atom.io[main Atom website] at atom.io or from the Atom project releases page at https://github.com/atom/atom/releases.
+To install Atom on Linux, you can download a https://atom.io/download/deb[Debian package] or https://atom.io/download/rpm[RPM package] either from the https://atom.io[main Atom website] at atom.io or from the Atom project releases page at https://github.com/atom/atom/releases. These packages do not have auto-update features, however, so whenever a new release of Atom comes out that you would like to upgrade to, you will have to repeat this installation process again for the new release. 
```

Comment:
Let's make this a bit more concise, how about "... do not currently have auto-update features, so when you would like to upgrade to a new release of Atom, you will have to repeat this installation process."